doc/compiler/runtime/J9JITExceptionTable.md

```diff
@@ -0,0 +1,400 @@
+<!--
+Copyright IBM Corp. and others 2023
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Background
+
+The `J9JITExceptionTable` (also `typedef`d to `TR_MethodMetaData`)
+is generally referred to as "the metadata". It contains most of the
+metadata associated with a compiled body (the other metadata
+is stored in the `TR_PersistentJittedBodyInfo`). Most of this metadata
+is stored in the variable length section of the `J9JITExceptionTable`
+structure. This document outlines this variable length section. It does
+not describe fields that is described by a datastructure (e.g., the
+`J9JITExceptionTable`, `TR_InlinedCallSite`, etc.).
+
+# Structure
+
+## High Level Overview
+
+```
++-----------------------------------+
+| struct J9JITExceptionTable        |
+|-----------------------------------|
+| Exception Info                    |
+|-----------------------------------|
+| Inline Info                       |
+|-----------------------------------|
+| Stack Atlas                       |
+|-----------------------------------|
+| Stack Alloc Map                   | // If Local Objects
+|-----------------------------------|
+| Internal Pointer Map in J9 Format |
+|-----------------------------------|
+| OSR Info                          | // If OSR
+|-----------------------------------|
+| GPU Info                          | // If GPU
+|-----------------------------------|
+| Bytecode PC to IA Map             | // If Hardware Profiling
++-----------------------------------+
+```
+
+## Exception Info
+The Exception Info consists of the following set of data for each exception
+in the method body
+```
++-------------------------------+
+| Instruction Start PC Offset   |
+| Size: 2 or 4 bytes            |
+|-------------------------------|
+| Instruction End PC Offset     |
+| Size: 2 or 4 bytes            |
+|-------------------------------|
```

Comment:
Just a question to help myself understand: Is the start to end PC offset the code that throws the instruction?